fast64_internal/oot/cutscene/classes.py

```diff
@@ -49,7 +49,7 @@ class CutsceneCmdCamPoint(CutsceneCmdBase):
     def __post_init__(self):
         if self.params is not None:
             self.continueFlag = self.params[0]
-            self.camRoll = getInteger(self.params[1])
+            self.camRoll = getInteger(self.params[1], not bpy.context.scene.fast64.oot.is_globalh_present())
```

Comment:
again not a fan of checking for global.h as a "is newer assets system"

what changed here in the assets?

Reply:
bruh

how did 0xFFFFFFE2 work previously?